output/html/db_callers.php

```diff
@@ -74,7 +74,7 @@ public function output() {
 					}
 				}
 
-				echo '<td class="qm-num">' . esc_html( $stime ) . '</td>';
+				echo '<td class="qm-num" data-qm-sort-weight="' . esc_attr( number_format( $row['ltime'], 4, '.', '' ) ) . '">' . esc_html( $stime ) . '</td>';
```

Comment:
Seeing as `$row['ltime']` here is an actual float, there's no need to pass it through `number_format()`. It can be used as-is.